content/humans/eileen-gu/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Eileen (Ailing) Gu"
+aka: []
+date: "2021-05-24T00:25:29"
```

Comment:
another thing to note is that the date allows `hugo` to generate the website with the _latest_ content at the top, so these new people added won't show up under "recent humans".

you can use `hugo` to generate these files!

Reply:
lolol, Thanks! I'll install Hugo